tests/test_benchmark/task_grid.py

```diff
@@ -55,19 +67,31 @@
 
 
 # Mock tasks for testing - intended to be faster and avoid downloading data leading to false positive potential failures in CI
-# Not all tasks are implemnted as Mock tasks yet
+# Not all tasks are implemented as Mock tasks yet
 MOCK_TASK_TEST_GRID = [
     MockBitextMiningTask(),
+    MockMultilingualBitextMiningTask(),
+    MockMultilingualParallelBitextMiningTask(),
     MockClassificationTask(),
+    MockMultilingualClassificationTask(),
     MockClusteringTask(),
+    MockMultilingualClusteringTask(),
     MockClusteringFastTask(),
+    MockMultilingualClusteringFastTask(),
     MockPairClassificationTask(),
+    MockMultilingualPairClassificationTask(),
     MockRerankingTask(),
+    MockMultilingualRerankingTask(),
     MockRetrievalTask(),
+    MockMultilingualRetrievalTask(),
     MockSTSTask(),
+    MockMultilingualSTSTask(),
     MockMultilabelClassification(),
+    MockMultilingualMultilabelClassification(),
     MockSummarizationTask(),
+    MockMultilingualSummarizationTask(),
     MockInstructionRetrival(),
+    MockMultilingualInstructionRetrival(),
```

Comment:
How does this influence runtime of the tests?

Reply:
I looked through CI runs and time nearly same. Locally all tasks run ~20 sec